samples/notebooks/python/09-streaming-completions.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
Since these two notebooks are similar, it may make sense to combine them into one. If you do so, I'd rename it to `09-streaming-and-multiple-completions`. I'll leave it up to you.

Reply:
I originally had them as a single notebook and it got a little out of hand. They have similar flow, but they are fundamentally different features that are hard to combine into a single, simple notebook.